fjord/alerts/models.py

```diff
@@ -69,6 +79,28 @@ def __unicode__(self):
         return self.url
 
 
+class UTCDateTimeField(fields.DateTimeField):
+    """Like DateTimeField, except it always in UTC"""
+    def to_native(self, value):
+        if value is not None and value.tzinfo is None:
+            default_tzinfo = pytz.timezone(settings.TIME_ZONE)
+            value = default_tzinfo.localize(value)
+            value = value.astimezone(pytz.utc)
+        return super(UTCDateTimeField, self).to_native(value)
+
+    def from_native(self, value):
+        result = super(UTCDateTimeField, self).from_native(value)
+        # We're using MySQL with settings.USE_TZ=False. So we need to
+        # peel off the timezone if we get sent one.
+        if result is not None and result.tzinfo is not None:
+            # First convert it to UTC
+            result = result.astimezone(pytz.UTC)
+
+            # Then drop the timezone
+            result = result.replace(tzinfo=None)
+        return result
```

Comment:
I'm not sure this is quite right. I think this will end up shifting dates by 8 hours every time (because 8 hours is Input's offset from UTC). I'm imagining this:

1. I decide that something happened, so I send call the alert API. Like a good user, I encode the timestamp in ISO8601, in the UTC time zone. Something like `2015-03-02T17:06:35Z`.
2. Line 92 converts it to a non-naive Python datetime object (in UTC, representing the right time). Something like `datetime.datetime(2015, 3, 2, 17, 6, 35, 0, tzinfo=<UTC>)`.
3. Line 97 converts it to UTC, which is a no-op.
4. Line 100 drops the time zone info. That will turn it from a non-naive datetime to a naive one with the same numbers, like `datetime.datetime(2015, 3, 2, 17, 6, 35, 0, tzinfo=None)`.
5. That goes in the DB.
6. Later, I want to know about that alert, so I ask the API. The timestamp is fetched as a naive datetime from the DB.
7. Line 87 converts the timestamp into a non-naive datetime object in PST without changing the numbers. Something like `datetime.datetime(2015, 3, 2, 17, 6, 35, 0, tzinfo=<DstTzInfo 'US/Pacific' PST-1 day, 16:00:00 STD>)`
8. Line 88 converts that to UTC, resulting in something like `datetime.datetime(2015, 3, 3, 1, 06, 35, 0, tzinfo=<UTC>)`. Importantly, this step changes the hour (and the day, but just because time wraps).
9. That datetime is serialized into a string to be sent out in JSON. It looks like `2015-03-03T01:06:35Z`. Not what we started with.

Does that logic sound correct? I think what you need to do is convert to `pytz.timezone(settings.TIME_ZONE)` on line 97, instead of UTC.